docs/dev/api/concrete.compiler.client_parameters.md

```diff
@@ -0,0 +1,128 @@
+<!-- markdownlint-disable -->
+
+<a href="../../tempdirectoryforapidocs/.venvtrash/lib/python3.10/site-packages/concrete/compiler/client_parameters.py#L0"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
```

Comment:
there are some ugly ../../tempdirectoryforapidocs/.venvtrash/lib/python3.10 which I could remove, but I don't think they have any impact

Reply:
Actually it had an impact and I fixed the links